releases/v1.0.1-rc.0.toml

```diff
@@ -0,0 +1,24 @@
+# commit to be tagged for new release
+commit = "HEAD"
```

Comment:
Does this work from a sub-branch?

Reply:
Yes, `HEAD` always points to the current branch reference.